src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResource.kt

```diff
@@ -100,12 +100,10 @@ class IepLevelResource(
   suspend fun getPrisonerIepLevelHistory(
     @Schema(description = "Booking Id", example = "3000002", required = true, type = "integer", format = "int64", pattern = "^[0-9]{1,20}$")
     @PathVariable bookingId: Long,
-    @Schema(description = "Use NOMIS data", example = "true", required = false, defaultValue = "true", hidden = true, type = "boolean", pattern = "^[true|false]$")
-    @RequestParam(defaultValue = "true", value = "use-nomis-data", required = false) useNomisData: Boolean = true,
```

Comment:
As mentioned this would be an API breaking change but since no consumers seem to explicitly pass/use this param (e.g. they default to getting the data from NOMIS) we think that's OK.

Reply:
correct